### DISCUSSION TITLE
Can you hack for me?

### DISCUSSION THREAD

**Discussion by KittyMasta** [Prodigy-Hacking/ProdigyMathGameHacking#858] [(A) Anything]
I need to find out how to hack using github can someone hack my prodigy account and tell me how to hack I'm new so I need help

**Comment by ArcerionDev** [comment]
https://github.com/Prodigy-Hacking/ProdigyMathGameHacking/discussions/170

**Comment by ghost** [comment]
The front screen literally says a discussion about this

**Reply by KittyMasta** [reply]
What front screen?

**Reply by ghost** [reply]
#170 click this

**Reply by KittyMasta** [reply]
Oh hehe i feel silly now ok so how exactly do you hack though i dont understand it all

**Reply by KittyMasta** [reply]
Sorry if im bugging you but im ten so i dont exactly understand

**Reply by SakuraKitsunee** [reply]
@KittyMasta is your computer a school computer or no?

**Reply by ArcerionDev** [reply]
2 things:
Please read instructions here: https://github.com/Prodigy-Hacking/ProdigyMathGameHacking/wiki/How-to-install-hacks
And, you need to be 13 years old to use Github, as shown [here](https://docs.github.com/en/github/site-policy/github-terms-of-service#b-account-terms).

**Reply by KittyMasta** [reply]
I have a amazon fire (:

**Reply by KittyMasta** [reply]
OH MY GOD THANK U SOOOO MUCH OH MY GOD OH MY GOD

**Reply by KittyMasta** [reply]
Wait the arrow is not there when will it appear does it appear after the prodigy tutorial?🤔

**Reply by SakuraKitsunee** [reply]
@KittyMasta its supposed to appear within 10 to 15 seconds once you login to the account

**Reply by KittyMasta** [reply]
Does it work on fire

**Reply by ghost** [reply]
@KittyMasta  ur banned

**Reply by NX721** [reply]
> @KittyMasta ur banned

?

**Reply by ghost** [reply]
@ArcerionDev

**Reply by ghost** [reply]
pls chat
and allso pls tell meh how to play prodigy on a school puter {my school blocks it}

**Reply by KittyMasta** [reply]
Type prodigy into google and press play the game

**Reply by ghost** [reply]
I NOW BUT MY SCHOOL BLOCKS IT

**Reply by KittyMasta** [reply]
Oh confusing!

**Reply by KittyMasta** [reply]
Can you download apks causs you could download prodigy apk

**Reply by KittyMasta** [reply]
Guys i need your help my tablet has a virus and notifacations and adds keep poping up!!!!!! I need your help!!!!!!!!!!!!!!!!!

**Reply by PatheticMustan** [reply]
???! How did you manage to get a virus

**Reply by NX721** [reply]
> ???! How did you manage to get a virus

Yea how?!

**Reply by ArcerionDev** [reply]
they probably looked for a different hacking site and it led to one of those "click allow notifications" scams

**Reply by ghost** [reply]
nope my school block the apk... unless u could paste the file here!!! then i could

**Reply by ghost** [reply]
yea like gamehack.icu

**Reply by ghost** [reply]
and patanegre.biz

**Reply by ghost** [reply]
sigh deleate any thig that look s supicus- downloads and scan yer ipad

**Reply by KittyMasta** [reply]
Yes it said allow notifacations and i click yes so how do i remove it

**Reply by KittyMasta** [reply]
https://apkpure.com/prodigy-math-game/com.prodigygame.prodigy

**Reply by KittyMasta** [reply]
Did you get prodigy?

**Reply by ghost** [reply]
nope i mean just litearly paste it here and when i click on it it downloads like that

**Reply by KittyMasta** [reply]
Dude thats what i did its up there

**Reply by ghost** [reply]
no it take me to a website that my school blocks

**Reply by KittyMasta** [reply]
Well then i guess i cant help you ):

**Reply by KittyMasta** [reply]
Leme think....

**Reply by KittyMasta** [reply]
You can use a different divice?
